### DISCUSSION TITLE
How to view and examine cells with extremely long content?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2650] [Q&A] [answered]
I have a CSV where teh content of a few cells (all in the same column) are ***very*** long, say 250 characters. Any way to scroll through the content of such a cell to examine it?

**Comment by saulpw** [comment, marked as answer]
You can use `v` to make the cursor row be multiline, or you can use `zl` and `zh` to scroll right and left through cell contents.

**Reply by halloleo** [reply]
Thanks @saulpw! `v` is great!

**Reply by halloleo** [reply]
Also, using `v` for a bit now, I'm wondering whether there is a way to say some sheets should always/usually open in `v` mode?

I noticed I usually only need the `v` mode on the sheet which opens when I press "Enter " on a normal row of the CSV (getting a transposed view of just that row -- very handy BTW!).

In short: Is it possible to give a sheet a "default format"?

**Comment by midichef** [comment]
Another option is to view/edit the cell inside your system's built-in `$EDITOR` by pressing `^O` (`sysedit-cell`). To try it, [install the latest `develop` branch](https://www.visidata.org/install/#update-visidata-from-an-existing-installation), as it is a new feature, and its behavior has just had an important bug fixed.
